modules/ossm-release-notes-3-0-new-features.adoc

```diff
@@ -0,0 +1,98 @@
+
+[id="revision-based-upgrades_{context}"]
+== Revision based upgrades
+
+This release introduces support for a `RevisionBased` upgrade strategy, which is based on the Istio canary control plane upgrade procedure. This enables a new Istio control plane to be created alongside the existing Istio control plane so that workloads may be migrated incrementally. This strategy is in addition to the `InPlace` upgrade strategy, which was the only supported approach for upgrading {SMProduct} 2.6 and earlier. The upgrade strategy is configured using the `spec.updateStrategy` field in the `Istio` resource.
+
+For more information, see "About RevisionBased strategy".
+
+[id="istiocni-customer-resource-definition_{context}"]
+== IstioCNI custom resource definition
+
+This release introduces the `IstioCNI` CRD, which is used to manage the lifecycle of the Istio Container Network Interface (CNI) DaemonSet. A single instance of this resource must be created per cluster to configure traffic redirection for pods in the mesh. The Istio CNI lifecycle is independent of the Istio control plane(s).
+
+[id="dual-stack-ipv4-ipv6_{context}"]
+== IPv4/IPv6 dual stack (Technology Preview)
+:FeatureName: IPv4/IPv6 dual stack
+include::snippets/technology-preview.adoc[]
+
+This release includes IPv4/IPv6 dual stack support as a technology preview feature. This aligns with the 'Alpha' status of the Istio upstream project, and is feature-complete for Istio when using sidecars. Dual stack helps organizations smoothly transition to IPv6, while still maintaining compatibility with their existing IPv4 setup.
+
+In {SMProduct} 3.0, dual stack is disabled by default. You can enable it with specific configuration changes, such as the ones shown in the following example:
+
+.Example configuration for IPv4/IPv6 Dual Stack
+[source,yaml]
+----
+spec:
+  values:
+    meshConfig:
+      defaultConfig:
+        proxyMetadata:
+          ISTIO_DUAL_STACK: "true"
+    pilot:
+      ipFamilyPolicy: RequireDualStack
+      env:
+        ISTIO_DUAL_STACK: "true"
+----
+
+[id="istio-ambient-mode_{context}"]
+== Istio Ambient mode (Developer Preview)
+:FeatureName: Istio Ambient mode
+include::snippets/developer-preview.adoc[]
+
+Istio Ambient Mode provides a sidecar-less service mesh architecture that reduces resource overhead, simplifies operations, and allows incremental adoption without application changes. It maintains security and observability through a layered security model with mTLS and authorization. OpenShift Service Mesh 3 operator includes deploying the Ambient profile as a developer preview feature using the community Ztunnel image. However, the ambient profile should not be used on clusters with production workloads or for multi-control plane use cases. Additionally, the community Ztunnel image does not work on IBM Power and Z Platforms.
```

Comment:
We should add to the list that the ZTunnel image will also not work on "OpenShift Container Platform clusters in FIPS mode"

Source: https://redhat-internal.slack.com/archives/C01R9E3SFEV/p1740762618836279?thread_ts=1740759024.029839&cid=C01R9E3SFEV